src/CycloneDX.Core/Models/Declarations/Signature.cs

```diff
@@ -0,0 +1,32 @@
+﻿// This file is part of CycloneDX Library for .NET
+//
+// Licensed under the Apache License, Version 2.0 (the “License”);
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an “AS IS” BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) OWASP Foundation. All Rights Reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CycloneDX.Models
+{
+    public class Signature
+    {
+        public string Algorithm { get; set; }
+        public string KeyId { get; set; }
+        public string PublicKey { get; set; }
```

Comment:
PublicKey is an object... However, in general I wonder whether there is no library to support the JSF.

Reply:
Should be fixed by https://github.com/CycloneDX/cyclonedx-dotnet-library/pull/356